password_security/__manifest__.py

```diff
@@ -5,7 +5,7 @@
 {
     "name": "Password Security",
     "summary": "Allow admin to set password security requirements.",
-    "version": "15.0.1.3.0",
+    "version": "15.0.1.3.1",
```

Comment:
No need to change the module version, as the bot does it on merge and you avoid possible merge conflicts due to several PRs modifying it.